### ISSUE TITLE
Georeferencer missing in QGIS 3.26.0 (OSGeo4W)

### ISSUE THREAD

**Issue by agiudiceandrea** [qgis/QGIS#49139]
### What is the bug or the crash?

The Georeferencer tool is missing in QGIS 3.26.0 on Windows 10 (OSGeo4W).
The tool is available in QGIS 3.22.8 (OSGeo4W) installed on the same system.

### Steps to reproduce the issue

1. Click on Raster menu: no Georeferencer... item is present

### Versions

QGIS version
3.26.0-Buenos Aires
QGIS code revision
0aece2818e
Qt version
5.15.3
Python version
3.9.5
GDAL/OGR version
3.5.0
PROJ version
9.0.1
EPSG Registry database version
v10.064 (2022-05-19)
GEOS version
3.10.3-CAPI-1.16.1
SQLite version
3.38.1
PDAL version
2.3.0
PostgreSQL client version
unknown
SpatiaLite version
5.0.1
QWT version
6.1.6
QScintilla2 version
2.13.1
OS version
Windows 10 Version 1903

Active Python plugins
db_manager
0.1.20
grassprovider
2.12.99
MetaSearch
0.3.6
processing
2.12.99
sagaprovider
2.12.99

### Supported QGIS version

- [X] I'm running a supported QGIS version according to the roadmap.

### New profile

- [X] I tried with a new QGIS profile

**Comment by uclaros** [comment]
Since support for vector layers has been integrated, it migrated to the `Layers` menu.
If it's still not there, maybe it lost its way :)

**Comment by agiudiceandrea** [comment]
> Since support for vector layers has been integrated, it migrated to the `Layers` menu.

Yes, it is in the crowded Layers menu. It is a documentation fault... Maybe better to put it in both Vector and Raster menu.

**Comment by gi921120** [comment]
> Since support for vector layers has been integrated, it migrated to the `Layers` menu. If it's still not there, maybe it lost its way :)

Hi there. The Georrefencer doesn't work properly in 3.26. I  tried with different transformations, but the output is sent to the wrong location. QGIS 3.22 does work correctly.

**Comment by agiudiceandrea** [comment]
@gi921120, could you please file another issue report with all the detailed info requested and all the steps and data to reproduce the issue?
